demo/src/lib/markdown/renderers/MdLink.svelte

```diff
@@ -0,0 +1,9 @@
+<script lang="ts">
+	export let href = '';
+	export let title: string;
+
+	const validMdRegex = /^\d{2}-[a-zA-Z-]*\.md$/g;
```

Comment:
This won't work if you reference another category.
Example: `(Focus Track)[../services/01-FocusTrack.md)`

Also, the extension is not mandatory in most of the markdown renderer; you can have both URL path and file path

Reply:
Thx for the suggestion, have updated the MdLink renderer to support relative markdown links